### PR TITLE
Deploy/aerodrome

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

**Comment by Thomas-Smets** [comment]
Still have to fill in the risk related parameters in the Constant file.

File: script/utils/Constants.sol [hunk 1]
```diff
@@ -0,0 +1,330 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.22;
+
+library ArcadiaContracts {
+    address internal constant AERO_POOL_AM = address(0xfe0FA1FD8F8E889062F03e2f126Fc7B9DE6091A5); // TODo:fork address!!
```
Comment (acollette) [inline script/utils/Constants.sol:8]: that's not fork address anymore ? Confusing comment

**Review by acollette — DISMISSED** [review]
lgtm

File: script/utils/Constants.sol (hunk 1 above)
Comment (Thomas-Smets) [reply script/utils/Constants.sol:8]: Yes it is still a fork address of that AM, but we can only change it after deploy step2 on mainnet

File: script/utils/Constants.sol [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+    uint16 internal constant COL_FAC_WSTETH_WETH = 9100;
+    uint16 internal constant COL_FAC_WSTETH_USDC = 8100;
+    uint16 internal constant LIQ_FAC_WSTETH_WETH = 9500;
+    uint16 internal constant LIQ_FAC_WSTETH_USDC = 9400;
+    uint112 internal constant EXPOSURE_WSTETH_WETH = uint112(400 * 10 ** Decimals.WSTETH);
+    uint112 internal constant EXPOSURE_WSTETH_USDC = uint112(300 * 10 ** Decimals.WSTETH);
+
+    // ToDo: change before deployment
+    uint16 internal constant COL_FAC_AERO_WETH = 8000;
+    uint16 internal constant COL_FAC_AERO_USDC = 8000;
```
Comment (zekiblue) [inline script/utils/Constants.sol:235]:
```suggestion
    uint16 internal constant COL_FAC_AERO_USDC = 6700;
```

File: script/utils/Constants.sol [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+    uint16 internal constant LIQ_FAC_AERO_WETH = 9000;
+    uint16 internal constant LIQ_FAC_AERO_USDC = 9000;
```
Comment (zekiblue) [inline script/utils/Constants.sol:237]:
```suggestion
    uint16 internal constant LIQ_FAC_AERO_USDC = 8000;
```

File: script/utils/Constants.sol [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+
+    uint16 internal constant COL_FAC_WSTETH_WETH = 9100;
+    uint16 internal constant COL_FAC_WSTETH_USDC = 8100;
+    uint16 internal constant LIQ_FAC_WSTETH_WETH = 9500;
+    uint16 internal constant LIQ_FAC_WSTETH_USDC = 9400;
+    uint112 internal constant EXPOSURE_WSTETH_WETH = uint112(400 * 10 ** Decimals.WSTETH);
+    uint112 internal constant EXPOSURE_WSTETH_USDC = uint112(300 * 10 ** Decimals.WSTETH);
+
+    // ToDo: change before deployment
+    uint16 internal constant COL_FAC_AERO_WETH = 8000;
```
Comment (zekiblue) [inline script/utils/Constants.sol:234]:
```suggestion
    uint16 internal constant COL_FAC_AERO_WETH = 7500;
```

File: script/utils/Constants.sol [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+    uint16 internal constant COL_FAC_AERO_USDC = 8000;
+    uint16 internal constant LIQ_FAC_AERO_WETH = 9000;
```
Comment (zekiblue) [inline script/utils/Constants.sol:236]:
```suggestion
    uint16 internal constant LIQ_FAC_AERO_WETH = 8700;
```

**Review by zekiblue — APPROVED** [review]

File: script/utils/Constants.sol [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+    uint16 internal constant LIQ_FAC_AERO_USDC = 9000;
+    uint112 internal constant EXPOSURE_AERO_WETH = uint112(1e6 * 10 ** Decimals.AERO);
```
Comment (zekiblue) [inline script/utils/Constants.sol:238]:
```suggestion
    uint112 internal constant EXPOSURE_AERO_WETH = uint112(3e6 * 10 ** Decimals.AERO);
```

File: script/utils/Constants.sol [hunk 7]
```diff
@@ -0,0 +1,330 @@
...
+    uint112 internal constant EXPOSURE_AERO_USDC = uint112(1e6 * 10 ** Decimals.AERO);
```
Comment (zekiblue) [inline script/utils/Constants.sol:239]:
```suggestion
    uint112 internal constant EXPOSURE_AERO_USDC = uint112(3e6 * 10 ** Decimals.AERO);
```

**Review by j-vp — APPROVED** [review]

**Review by zekiblue — APPROVED** [review]

**Pull request merged**